config/default.yml

```diff
@@ -83,6 +83,15 @@ Rails/ActiveSupportAliases:
   Enabled: true
   VersionAdded: '0.48'
 
+Rails/AddColumnComment:
+  Description: >-
+    This cop enforces the use of the `comment` option when creating a new column
+    during a migration by calling either `add_column`, `t.column` or any other
+    alias method such as `t.integer`, `t.string` etc.
+  Enabled: false
+  Safe: false
```

Comment:
What is the reason why this cop is unsafe?

Reply:
No reason, but I saw the discussion on #453 and thought it was fair to deploy it as `unsafe` first, but I don't think there is anything unsafe about it :)